src/schematools/contrib/django/app_config.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+
+from django.apps import apps
+from django.apps import config as apps_config
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class VirtualAppConfig(apps_config.AppConfig):
+    """
+    Virtual App Config, allowing to add models for datasets on the fly.
+    """
+
+    def __init__(self, apps, app_name, app_module):
```

Comment:
As we're trying to get things fully type annotated in `schematools` the members of this class could do with type annotations.